daemon/config/config.go

```diff
@@ -44,7 +44,12 @@ const (
 	DisableNetworkBridge = "none"
 	// DefaultInitBinary is the name of the default init binary
 	DefaultInitBinary = "docker-init"
-
+	// DefaultShimBinary is the default shim to be used by containerd if none
+	// is specified
+	DefaultShimBinary = "containerd-shim"
```

Comment:
Not a new in this PR, but v1 shim is no longer used by default. Can we add a godoc about that?

Reply:
Hmm.. was about to update the description, but looks like it's still used if `containerd` is started as child process;  https://github.com/moby/moby/blob/48da116486059e57e1962234d6751b4b977ee1c8/cmd/dockerd/daemon_unix.go#L64-L73


Should that be updated to `containerd-shim-runc-v2` ?

Reply:
Or should it be removed altogether? I see that if the containerd-config looks like this;

```toml
[plugins]
  [plugins.linux]
    shim = "containerd-shim"
    runtime = "runc"
    runtime_root = "/var/lib/docker/runc"
    no_shim = false
    shim_debug = true
```

Containers are still started with `containerd-shim-runc-v2`, so looks like the configuration is not needed anymore?

I see containerd also sets `containerd-shim` and `runc` as defaults; https://github.com/containerd/containerd/blob/9b4967bd6b77e4a6bb4c36c86b4e3a6daaead6d4/runtime/v1/linux/runtime.go#L71-L82

So perhaps it should be removed, or is there still a path where this would be used?